README.rst

```diff
@@ -49,6 +49,28 @@ Get involved!
 -  Read more about the project's motivation at `Introducing KA Lite, an offline version of Khan
    Academy <http://jamiealexandre.com/blog/2012/12/12/ka-lite-offline-khan-academy/>`__.
 
+Roadmap
+-------
+
+During 2017, Learning Equality is launching the successor of KA Lite. It's
+called `Kolibri <http://github.com/learningequality/kolibri>`__ and will have
```

Comment:
Wondering whether it might be better to link to https://learningequality.org/kolibri/ for now, to avoid ambitious people trying to install from Github and then being disappointed (no channels, etc).